### DISCUSSION TITLE
Web player poor audio quality

### DISCUSSION THREAD

**Discussion by eros404** [navidrome/navidrome#3503] [Q&A]
Hello,

I did not see any mention of this issue, i do not know if it's the right place to talk about this.

Just wanted to say that actually (0.53.3) the web player audio output quality is really not satisfying. To be more precise it feels really flat.

My setup is a Rasperry Pi (with Docker) and I stream a bunch of FLAC files, I also have Jellyfin deployed and its web player really gives a better audio experience.

So it's there somebody out there that experienced the same thing ?
Maybe there is some transcoding options that I could tweek....

**Comment by metalheim** [comment]
Navidrome really only passes the audio files to the Browser and the Browser engine plays the audio. 

Are you sure it's not one of these things?
1. loudness (quiet music is perceived as sounding less good) 
Since navidrome uses replayGain, it could be that navidrome plays your songs more quiet

2. Are you transcoding? If yes, turn off transcoding? Most browser should be able to play FLAC just fine, as long as you dont have any special highres files or more than 2 audio streams

If youre sure navidrome still sounds worse, then maybe try a different browser (again, the browser plays the file, not navidrome).
Alternatively, you can try other clients like https://github.com/jeffvli/feishin. But be aware that most Browser-based client just leave the playback to the browser and thus will sound the same.
